src/node/index.ts

```diff
@@ -423,29 +433,32 @@ export default function PluginInspect(options: Options = {}): Plugin {
       }
     }
 
-    if (!silent) {
-      const _print = server.printUrls
-      server.printUrls = () => {
-        const colorUrl = (url: string) => c.green(url.replace(/:(\d+)\//, (_, port) => `:${c.bold(port)}/`))
+    const _print = server.printUrls
+    server.printUrls = () => {
+      const colorUrl = (url: string) => c.green(url.replace(/:(\d+)\//, (_, port) => `:${c.bold(port)}/`))
 
-        let host = `${config.server.https ? 'https' : 'http'}://localhost:${config.server.port || '80'}`
+      let host = `${config.server.https ? 'https' : 'http'}://localhost:${config.server.port || '80'}`
 
-        const url = server.resolvedUrls?.local[0]
+      const url = server.resolvedUrls?.local[0]
 
-        if (url) {
-          try {
-            const u = new URL(url)
-            host = `${u.protocol}//${u.host}`
-          }
-          catch (error) {
-            console.warn('Parse resolved url failed:', error)
-          }
+      if (url) {
+        try {
+          const u = new URL(url)
+          host = `${u.protocol}//${u.host}`
+        }
+        catch (error) {
+          console.warn('Parse resolved url failed:', error)
         }
+      }
 
+      if (!silent) {
         _print()
         // eslint-disable-next-line no-console
         console.log(`  ${c.green('➜')}  ${c.bold('Inspect')}: ${colorUrl(`${host}${base}__inspect/`)}`)
       }
+      // I can't get the config when the service starts.
```

Comment:
What do you mean by this line?

Reply:
I’ve try implmenet the logic outside of `printUrls`. But vite cann't get `config` during startup.So I can't get the correct route.Should i put the comment inside `printUrls` right?